### ISSUE TITLE
2.150.0 You must provide a value for the attribute 'contentRightsDeclaration' with this request

### ISSUE THREAD

**Issue by thinklinux** [fastlane/fastlane#16727]
### New Release Candidate Issue Checklist

- [x] Updated fastlane to the latest release candidate version (2.150.0)
- [ ] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)
* Breaking version: [`2.150.0`]

* Last working version: [`2.149.1`]

* State of app: [`Prepare for Submission`]

* First version of app?: [`Yes`]
Issue Description

The problem occurs at the submit_for_review stage.

This is a new app, and everything was fine until submit for review step. Then the following error occurred:

The request cannot be fulfilled because of the state of another resource. - Submit for review errors found.
The provided entity is missing a required attribute - You must provide a value for the attribute 'contentRightsDeclaration' with this request
### Mentioning

cc: @joshdholtz

<details><summary>🚫 fastlane environment 🚫</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.15.4                                     |
| Ruby                        | 2.6.5                                       |
| Bundler?                    | true                                        |
| Git                         | git version 2.24.3 (Apple Git-128)          |
| Installation Source         | ~/.rbenv/versions/2.6.5/bin/fastlane        |
| Host                        | Mac OS X 10.15.4 (19E266)                   |
| Ruby Lib Dir                | ~/.rbenv/versions/2.6.5/lib                 |
| OpenSSL Version             | OpenSSL 1.1.1d  10 Sep 2019                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 11.5                                        |

### System Locale

| Error                       |
| --------------------------- |
| No Locale with UTF8 found 🚫 |

### fastlane files:

**No Fastfile found**

**No Appfile found**

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.150.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.3.0        |
| bundler                   | 1.17.2       |
| etc                       | 1.0.1        |
| forwardable               | 1.2.0        |
| CFPropertyList            | 3.0.2        |
| public_suffix             | 4.0.5        |
| addressable               | 2.7.0        |
| atomos                    | 0.1.3        |
| aws-eventstream           | 1.1.0        |
| aws-partitions            | 1.337.0      |
| aws-sigv4                 | 1.2.1        |
| jmespath                  | 1.4.0        |
| aws-sdk-core              | 3.103.0      |
| aws-sdk-kms               | 1.35.0       |
| aws-sdk-s3                | 1.72.0       |
| babosa                    | 1.0.3        |
| claide                    | 1.0.3        |
| colored                   | 1.2          |
| colored2                  | 3.1.2        |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| declarative               | 0.0.20       |
| declarative-option        | 0.1.0        |
| digest-crc                | 0.6.0        |
| unf_ext                   | 0.0.7.7      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20190701 |
| dotenv                    | 2.7.5        |
| emoji_regex               | 3.0.0        |
| excon                     | 0.75.0       |
| multipart-post            | 2.0.0        |
| faraday                   | 1.0.1        |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 1.0.0        |
| fastimage                 | 2.1.7        |
| gh_inspector              | 1.1.3        |
| jwt                       | 2.1.0        |
| memoist                   | 0.16.2       |
| multi_json                | 1.14.1       |
| os                        | 1.1.0        |
| signet                    | 0.14.0       |
| googleauth                | 0.13.0       |
| httpclient                | 2.8.3        |
| mini_mime                 | 1.0.2        |
| uber                      | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.2        |
| google-api-client         | 0.38.0       |
| google-cloud-env          | 1.3.2        |
| google-cloud-errors       | 1.0.1        |
| google-cloud-core         | 1.5.0        |
| google-cloud-storage      | 1.26.2       |
| json                      | 2.3.1        |
| mini_magick               | 4.10.1       |
| multi_xml                 | 0.6.0        |
| plist                     | 3.5.0        |
| rubyzip                   | 2.3.0        |
| security                  | 0.1.3        |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.8        |
| slack-notifier            | 2.3.2        |
| terminal-notifier         | 2.0.0        |
| unicode-display_width     | 1.7.0        |
| terminal-table            | 1.8.0        |
| tty-screen                | 0.8.0        |
| tty-cursor                | 0.7.1        |
| tty-spinner               | 0.9.3        |
| word_wrap                 | 1.0.0        |
| nanaimo                   | 0.2.6        |
| xcodeproj                 | 1.17.0       |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| xcpretty-travis-formatter | 1.0.0        |
</details>

*generated on:* **2020-07-02**
</details>

<details>
<summary>Stack Trace</summary>

```
You can ⌘ + double-click on links to open them directly in your browser.
bundler: failed to load command: fastlane (/Users/vagrant/.rbenv/versions/2.6.5/bin/fastlane)
Spaceship::UnexpectedResponse: [!] The request cannot be fulfilled because of the state of another resource. - Submit for review errors found.
The provided entity is missing a required attribute - You must provide a value for the attribute 'contentRightsDeclaration' with this request
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/spaceship/lib/spaceship/connect_api/client.rb:164:in `handle_response'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/spaceship/lib/spaceship/connect_api/client.rb:100:in `post'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/spaceship/lib/spaceship/connect_api/tunes/tunes.rb:796:in `post_app_store_version_submission'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/spaceship/lib/spaceship/connect_api/models/app_store_version.rb:154:in `create_app_store_version_submission'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/deliver/lib/deliver/submit_for_review.rb:28:in `submit!'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/deliver/lib/deliver/runner.rb:172:in `submit_for_review'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/deliver/lib/deliver/runner.rb:49:in `run'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/actions/upload_to_app_store.rb:17:in `run'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/runner.rb:261:in `block (2 levels) in execute_action'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/runner.rb:253:in `block in execute_action'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/runner.rb:227:in `chdir'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/runner.rb:227:in `execute_action'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/runner.rb:157:in `trigger_action_by_name'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/fast_file.rb:159:in `method_missing'
  Fastfile:55:in `block (2 levels) in parsing_binding'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/lane.rb:33:in `call'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/runner.rb:45:in `execute'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/lane_manager.rb:47:in `cruise_lane'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/commands_generator.rb:108:in `block (2 levels) in run'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/commands_generator.rb:352:in `run'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/commands_generator.rb:41:in `start'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/fastlane/lib/fastlane/cli_tools_distributor.rb:119:in `take_off'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fastlane-2.150.0/bin/fastlane:23:in `<top (required)>'
  /Users/vagrant/.rbenv/versions/2.6.5/bin/fastlane:23:in `load'
  /Users/vagrant/.rbenv/versions/2.6.5/bin/fastlane:23:in `<top (required)>'
```
</details>

#16645

**Comment by petekanev** [comment]
@thinklinux 

This looks like a new requirement from the AppStoreConnect API which is already handled in the 2.150.0 release, but it remained undocumented - https://github.com/fastlane/fastlane/blob/master/deliver/lib/deliver/submit_for_review.rb#L174

Try providing `content_rights_contains_third_party_content: true/false` (depending on your app) in `submission_information` if using `fastlane deliver`.

E.g
```ruby
deliver(
   ...
   submission_information: {
          add_id_info_uses_idfa: false,
          export_compliance_compliance_required: false,
          export_compliance_encryption_updated: false,
          export_compliance_app_type: nil,
          export_compliance_uses_encryption: false,
          export_compliance_is_exempt: false,
          export_compliance_contains_third_party_cryptography: false,
          export_compliance_contains_proprietary_cryptography: false,
          content_rights_contains_third_party_content: false
        })
```

**Comment by joshdholtz** [comment]
@petekanev Thanks for answering! I had to go to the dentist so I didn't get a chance to this morning 🤷‍♂️ But...

@thinklinux 
I will add a helpful error message that explains how to add this when this occurs so there isn't any confusion 😊  I did this with export compliance and IDFA but not content right I guess 😬 I will get this fixed for 2.150.2 💪

**Comment by thinklinux** [comment]
Thank you both for the fast reaction!!! Closing this one ^^
